client/webserver/site/src/js/wallets.ts

```diff
@@ -450,10 +450,10 @@ export default class WalletsPage extends BasePage {
         tmpl.priceBaseUnit.textContent = basesymbol.toUpperCase()
         tmpl.volume.textContent = fourSigFigs(spotVolume(assetID, mkt))
         tmpl.volumeUnit.textContent = assetID === baseid ? basesymbol.toUpperCase() : quotesymbol.toUpperCase()
-      } else Doc.hide(tmpl.priceBox, tmpl.volumeBox)
+        page.marketsOverviewBox.classList.remove('invisible')
+      }
       Doc.bind(row, 'click', () => app().loadPage('markets', { host, base: baseid, quote: quoteid }))
     }
-    page.marketsOverviewBox.classList.remove('invisible')
```

Comment:
Why should the `marketsOverviewBox` depend on the spot prices being available?  We still want the markets listed whether or not the price and volume are available.

Reply:
No reason. So it should be fine like that:

![image](https://user-images.githubusercontent.com/15069783/186228152-ae2089f0-b7de-40df-8510-b4ecd8a89be9.png)